workers/scanner/src/main/kotlin/scanner/ScanSummaryHash.kt

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
Why create a full featured hash function instead of an `equals` one ?

You don't store the hash anywhere and you'd save on memory because you would not need to build a huge string.

Reply:
The hashes were used because again different types (ORT model and ORT Server model) are involved. But you are right, this is not the most efficient approach. Will try to come up with a better solution.

Reply:
I have added a comparison function that should be more efficient.